Client/Frontend/NTP/NTPNotificationView.swift

```diff
@@ -0,0 +1,151 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import Shared
+import BraveShared
+import BraveRewardsUI
+
+struct NTPNotificationViewConfig {
+    var headerText: String?
+    var bodyText: (text: String, urlInfo: [String: String], action: ((URL) -> Void)?)?
+    var primaryButtonConfig: (text: String, showCoinIcon: Bool, action: (() -> Void))?
+    var secondaryButtonConfig: (text: String, action: (() -> Void))?
+    let textColor: UIColor
+    
+    init(textColor: UIColor) {
+        self.textColor = textColor
+    }
+}
+
+class NTPNotificationView: UIStackView {
+    
+    private lazy var titleStackView = UIStackView().then {
```

Comment:
These shouldn't need to be lazy vars, using `let` should be fine.

Reply:
They are, because each of those views doesn't need to be added to the main view.

Some views have headers and buttons, some don't